packages/trader/src/Modules/Contract/Components/AccumulatorsStats/ticks-history-counter.tsx

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-const TicksHistoryCounter = ({ has_progress_dots, value }) => (
+type TTicksHistoryCounter = {
+    data_testid_progress_dots?: string;
+    data_testid_ticks_history_counter?: string;
+    has_progress_dots: boolean;
+    value: number;
+};
+
+const TicksHistoryCounter = ({
+    data_testid_progress_dots,
+    data_testid_ticks_history_counter,
+    has_progress_dots,
+    value,
+}: TTicksHistoryCounter) => (
     <div
-        data-testid='dt_accu_stats_history_counter'
+        data-testid={data_testid_ticks_history_counter}
```

Comment:
```suggestion
        data-testid='dt_ticks_history_counter'
```
@henry-deriv nit: data-testid start with 'dt_... ' according to convention on the project🙂